Class-Code/Elementary-CA/fcsmakers/sketch.js

```diff
@@ -11,4 +11,6 @@ function setup() {
 
 function draw() {
     background(0);
+    fill(255)
+    ellipse(frameCount,height/2,20,20)
```

Comment:
This needs bigger. 20 px just doesn't show up well. however good use of the frameCount Variable